pkg/test/options.go

```diff
@@ -0,0 +1,28 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import "github.com/aws/karpenter/pkg/utils/options"
+
+func Options() options.Options {
```

Comment:
Why is this broken out now? A lot of these are AWS specific flags, seems better to keep in the specific test suite and just reset or deep copy on each execution.

Reply:
Good catch, I forgot that these could be highly AWS-specific